flow/react-navigation.js

```diff
@@ -209,9 +239,8 @@ declare module 'react-navigation' {
     params?: NavigationParams,
   };
 
-  declare export type NavigationStateRoute =
-    & NavigationLeafRoute
-    & NavigationState;
+  declare export type NavigationStateRoute = NavigationLeafRoute &
```

Comment:
I find this reads less clearly than the original format, and it doesn't do as good of a job at preserving `git blame` attribution.

Reply:
I don't really control what prettier does here. Maybe we are using different versions or configurations of prettier?